colorcoder/Color.java

```diff
@@ -0,0 +1,6 @@
+package colorcoder;

```

Comment:
what is the importance of Color interface ?

Reply:
To remove duplication of method fromIndex(). Implementing a common interface allowed me to use a single fromIndex() method from which I could get Major and Minor colors using typecasting. 

I would have gone for a Color superclass so that getIndex() and toString() methods were also not repeated, but since I used enum it could not extend any other class due to multiple inheritances restriction.